src/pages/registration/RegistrationFormActions.tsx

```diff
@@ -69,6 +69,7 @@ export const RegistrationFormActions = ({
       } else {
         const newErrors = validateForm(updateRegistrationResponse.data);
         setTouched(setNestedObjectValues(newErrors, true));
+        resetForm({ values: updateRegistrationResponse.data, errors: newErrors });
```

Comment:
Tror jeg fant en bug. For å reprodusere:
1. Gå til "Nytt resultat"
2. Velg "Start i tomt registreringsskjema"
3. (Bruker skal nå bare valideringsfeil på faner man allerede har besøket. Trykker man "Lagre" skal man se alle valideringsfeil)
4. Når jeg trykker lagre nå så blir alle faner markert OK (blå)

Sikkert fordi vi her nullstiller `touched` rett etter at vi setter det?

```suggestion
        resetForm({ values: updateRegistrationResponse.data, errors: newErrors, touched: setNestedObjectValues(newErrors, true) });
```